README.md

```diff
@@ -0,0 +1,36 @@
+# to-do-list
+
```

Comment:
- [ ] The project has unnecessary files such as .vscode. Let's add that to .gitignore.